src/sentry/tasks/delete_seer_grouping_records.py

```diff
@@ -58,7 +58,8 @@ def call_delete_seer_grouping_records_by_hash(
             "calling seer record deletion by hash",
             extra={"project_id": project.id, "hashes": group_hashes},
         )
-        delete_seer_grouping_records_by_hash.apply_async(args=[project.id, group_hashes, 0])
+        if group_hashes:
+            delete_seer_grouping_records_by_hash.apply_async(args=[project.id, group_hashes, 0])
```

Comment:
Just curious: in what scenario would there ever not be hashes?

Reply:
this shouldn't happen anymore, and I'll remove this check once I confirm that the log above never shows no hashes